src/test/ui/const-generics/issue-70408.rs

```diff
@@ -1,4 +1,6 @@
+// revisions: normal mir
 // build-pass
+//[mir] compile-flags: -Zunpretty=mir
```

Comment:
IMO a separate test should be added. It is not super clear from looking at #70408 why this unpretty is necessary or why it is relevant to what’s described in the issue.

Reply:
I can split it out, I just didn't want to duplicate the file.